src/view/xl-label.inc.twig

```diff
@@ -0,0 +1,9 @@
+<div class="tooltip-html">
+  <button class="btn p-0 m-0"><i class="fa-solid fa-circle-info"></i></button>
```

Comment:
The button element should have something other than the class "fa-circle-info" to indicate to screen readers that this is an info button. For example, `<button class="btn p-0 m-0" aria-label="More info on this label">` or similar. Furthermore, it could maybe have something like `aria-controls="id-for-dl-element"`.